contracts/oracle/collateralization/CollateralizationOracleWrapper.sol

```diff
@@ -152,13 +154,27 @@ contract CollateralizationOracleWrapper is Timed, ICollateralizationOracleWrappe
             bool _validityStatus
         ) = ICollateralizationOracle(collateralizationOracle).pcvStats();
 
+        outdated = outdated 
+            || _isExceededDeviationThreshold(cachedProtocolControlledValue, _protocolControlledValue)
+            || _isExceededDeviationThreshold(cachedUserCirculatingFei, _userCirculatingFei);
+        
         // only update if valid
         require(_validityStatus, "CollateralizationOracleWrapper: CollateralizationOracle is invalid");
 
+        _setCache(_protocolControlledValue, _userCirculatingFei, _protocolEquity);
+
+        return outdated;
+    }
+
+    function _setCache(
+        uint256 _cachedProtocolControlledValue,
+        uint256 _cachedUserCirculatingFei,
+        int256 _cachedProtocolEquity
```

Comment:
Nit: these variables are not cached, they are new values to store in the cache. Recommend changing their names.